### ISSUE TITLE
how to send the all types file?and server side save the file as the client send name? 

### ISSUE THREAD

**Issue by saravanan-vdg** [simonguest/android-btxfr#6]
how to send the all types file?and server side save the file as the client send name?

**Comment by poralcode** [comment]
I agree. @simonguest  , do you have any example for sending and receiving any types of data, such pictures, files, text or string? That will be very helpful if there is.

**Comment by simonguest** [comment]
Have you seen [this repo](https://github.com/simonguest/BTPhotoTransfer-sample) - it contains an example client and server that sends photos from one device to another.

**Comment by poralcode** [comment]
yeah, but what about the other types of file? or simple string?

**Comment by simonguest** [comment]
Take a look at [this code](https://github.com/simonguest/btphototransfer-sample/blob/master/src/com/simonguest/BTPhotoTransfer/MainActivity.java#L189) in the MainActivity of the application. This is where the client loads the file that has just been taken using the camera. Should be pretty easy to replace this with opening a file from another location, or creating a new byte array if you want to send other types of data. Hope this helps!
